src/rp2_common/pico_cyw43_driver/CMakeLists.txt

```diff
@@ -155,4 +155,104 @@ if (EXISTS ${PICO_CYW43_DRIVER_PATH}/${CYW43_DRIVER_TEST_FILE})
     endfunction()
 
     pico_promote_common_scope_vars()
+
+    # Set env var, so it can be accessed in the function
+    set(ENV{PICO_CYW43_DRIVER_CURRENT_PATH} ${CMAKE_CURRENT_LIST_DIR})
```

Comment:
Why does this have to be an "ENV" var rather than just a normal CMake variable?

Reply:
It needs to be accessible from where the function is called (which is in the user's CMakeLists.txt) so a normal CMake variable wouldn't work as it doesn't get propogate upwards - but actually a pico_common_scope_var would probably be better than an environment variable so I'll change it to that